drone-kustomize/plugin.sh

```diff
@@ -27,5 +27,5 @@ fi
 kustomize build | kubectl apply -f -
 
 if [ -n "$PLUGIN_ROLLOUT_DEPLOYMENT" ];then
-  kubectl rollout status deployment $PLUGIN_ROLLOUT_DEPLOYMENT -n $PLUGIN_ROLLOUT_NAMESPACE --timeout=180s
-fi
\ No newline at end of file
+  kubectl rollout status deployment $PLUGIN_ROLLOUT_DEPLOYMENT -n $PLUGIN_ROLLOUT_NAMESPACE --timeout=${PLUGIN_ROLLOUT_TIMEOUT:-"180s"}
```

Comment:
Default value should be `0s` to don't disrupt the current behavior. After that, LGTM!

Reply:
Uh, previously `180s` was hardcoded. Are you asking me to put `0s` as the default fallback?

Reply:
OK! Sorry! You're right. LGTM!